cmd/acra-keys/keys/export_test.go

```diff
@@ -0,0 +1,274 @@
+	"github.com/cossacklabs/acra/keystore/keyloader"
+	"github.com/cossacklabs/acra/keystore/keyloader/env_loader"
+)
+
+func TestExport_Import_CMD_FS_V1(t *testing.T) {
+	clientID := []byte("testclientid")
+	keyloader.RegisterKeyEncryptorFabric(keyloader.KeystoreStrategyEnvMasterKey, env_loader.NewEnvKeyEncryptorFabric(keystore.AcraMasterKeyVarName))
+
+	masterKey, err := keystore.GenerateSymmetricKey()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	flagSet := flag.NewFlagSet(CmdMigrateKeys, flag.ContinueOnError)
+	keyloader.RegisterCLIParametersWithFlagSet(flagSet, "", "")
+
+	err = flagSet.Set("keystore_encryption_type", keyloader.KeystoreStrategyEnvMasterKey)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Setenv(keystore.AcraMasterKeyVarName, base64.StdEncoding.EncodeToString(masterKey))
+
+	keyStoreEncryptor, err := keyloader.CreateKeyEncryptor(flagSet, "")
+	if err != nil {
+		t.Fatal("Can't init keystore KeyEncryptor")
+	}
+
+	var (
+		keysFile = "access-keys.txt"
+		dataFile = "keys.dat"
+	)
+
+	t.Run("export/import storage private key", func(t *testing.T) {
+		defer func() {
+			if r := recover(); r != nil {
+				t.Errorf("Expected no panics in command")
+			}
+		}()
+
+		exportDirName := t.TempDir()
+		if err := os.Chmod(exportDirName, 0700); err != nil {
+			t.Fatal(err)
+		}
+
+		importDirName := t.TempDir()
+		if err := os.Chmod(importDirName, 0700); err != nil {
+			t.Fatal(err)
+		}
+
+		exportBackuper, err := filesystem.NewKeyBackuper(exportDirName, exportDirName, &filesystem.DummyStorage{}, keyStoreEncryptor)
+		if err != nil {
+			t.Fatal("Can't initialize backuper")
+		}
+
+		exportCMD := &ExportKeysSubcommand{
+			CommonKeyStoreParameters: CommonKeyStoreParameters{
+				keyDir: exportDirName,
+			},
+			CommonExportImportParameters: CommonExportImportParameters{
+				exportKeysFile: filepath.Join(exportDirName, keysFile),
+				exportDataFile: filepath.Join(exportDirName, dataFile),
+			},
+			exportIDs:     []string{"testclientid_storage"},
+			FlagSet:       flagSet,
+			exportPrivate: true,
+			exporter:      exportBackuper,
+		}
+
+		store, err := openKeyStoreV1(exportCMD)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		err = store.GenerateDataEncryptionKeys(clientID)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		ExportKeysCommand(exportCMD)
+
+		importBackuper, err := filesystem.NewKeyBackuper(importDirName, importDirName, &filesystem.DummyStorage{}, keyStoreEncryptor)
+		if err != nil {
+			t.Fatal("Can't initialize backuper")
+		}
+
+		importCMD := &ImportKeysSubcommand{
+			CommonKeyStoreParameters: CommonKeyStoreParameters{
+				keyDir: importDirName,
+			},
+			CommonExportImportParameters: CommonExportImportParameters{
+				exportKeysFile: filepath.Join(exportDirName, keysFile),
+				exportDataFile: filepath.Join(exportDirName, dataFile),
+			},
+			FlagSet:  flagSet,
+			importer: importBackuper,
+		}
+
+		ImportKeysCommand(importCMD)
+
+		_, err = os.ReadFile(filepath.Join(importDirName, "testclientid_storage"))
```

Comment:
let's use keystore targetted to this folder and try to read this file? file can exists but can be imported or encrypted incorrectly and keystore couldn't to read it